pandas/core/generic.py

```diff
@@ -5799,6 +5800,112 @@ def infer_objects(self: FrameOrSeries) -> FrameOrSeries:
             )
         ).__finalize__(self)
 
+    def convert_dtypes(
+        self: FrameOrSeries,
+        use_nullable_dtypes: bool_t = True,
+        convert_integer: bool_t = True,
```

Comment:
I suppose adding copy=False is ok, meaning we wouldn't make a copy if everything is already converted.

Reply:
I'm going to leave it for now as *always* doing a copy, because adding `copy=False` would make the logic more complicated.